src/openklant/components/utils/expansion.py

```diff
@@ -168,9 +174,19 @@ def inclusions_dict(self, serializer: Serializer) -> dict:
         entries = self._inclusions((), serializer, serializer.instance)
 
         for obj, inclusion_serializer, parent, path, many in entries:
-            data = inclusion_serializer(instance=obj, context=serializer.context).data
+            if not obj:
+                data = None
+                id = str(uuid4())
```

Comment:
Is there a reason why we need to fill id, when there is no object? From what I can quickly see locally this isn't necessary.